.gitignore

```diff
@@ -1,8 +1,6 @@
-.idea/
-.vscode/
 *.iml
 .env
 .DS_Store
-venv/
+*venv/
```

Comment:
The pattern `*venv/` might not ignore all virtual environment directories as expected. Consider using `venv/` or `*/venv/` to ensure all virtual environment directories are ignored, regardless of their location in the project.